pkg/trace/obfuscate/sql.go

```diff
@@ -167,12 +196,12 @@ func (f *groupingFilter) Reset() {
 // in strings and numbers by redacting them.
 func (o *Obfuscator) ObfuscateSQLString(in string) (*ObfuscatedQuery, error) {
 	lesc := o.SQLLiteralEscapes()
-	tok := NewSQLTokenizer(in, lesc)
+	tok := NewSQLTokenizer(in, lesc, o.sqlNewNormalization)
 	out, err := attemptObfuscation(tok)
 	if err != nil && tok.SeenEscape() {
 		// If the tokenizer failed, but saw an escape character in the process,
 		// try again treating escapes differently
-		tok = NewSQLTokenizer(in, !lesc)
+		tok = NewSQLTokenizer(in, !lesc, o.sqlNewNormalization)
```

Comment:
Please no more function `bool` arguments. I started this bad practice already 😅 You can just call `config.HasFeature` from anywhere you like. No need to pass stuff around.